test/test_discord_controller.py

```
@@ -0,0 +1,44 @@
+        try:
+            self.channel_id = int(config['all']['discord_channel_id'])
+            logger.info(f"TestDiscordController initialized: channel_id={self.channel_id}, hadInit={self.controller.hadInit}")
+        except (KeyError, ValueError):
+            self.skipTest("Invalid config.json: discord_channel_id or discord_token missing or invalid")
+        if not self.controller.hadInit:
+            self.skipTest("DiscordController not initialized")
+
+
+    async def test_recording_successful_todiscord(self):
+        title = "テスト番組"
+        message = f"\n{title} を録音しました!"
```

Ruff (0.8.0)

Local variable `message` is assigned to but never used

Remove assignment to unused variable `message`

(F841)